modules/analysis-common/src/yamlRestTest/resources/rest-api-spec/test/search.query/80_persian_search.yml

```diff
@@ -0,0 +1,44 @@
+# integration tests for persian analyzer changes from Lucene 9 to Lucene 10
+setup:
```

Comment:
If you want to test with old data, then upgrade, then verify the query results don't change, a rolling upgrade test, or one of the full restart tests.

Reply:
Yes I was afraid I'd need that full-blown infra and was somehow hoping I could leverage some yaml test. This at least shows that stemming works in the new version of the analyzer, i.e. both search terms are matching both documents which means they are analyzed to the same root form.